api/src/main/java/com/exelaration/abstractmemery/services/FileStorageService.java

```diff
@@ -0,0 +1,8 @@
+package com.exelaration.abstractmemery.services;
+
+import com.exelaration.abstractmemery.domains.Image;
+import org.springframework.web.multipart.MultipartFile;
+
+public interface FileStorageService {
+  public Image save(MultipartFile file);
```

Comment:
This should return something other than an `Image` object (`bool`, `string`, etc.).  It shouldn't care that it's saving an image, just that it's saving some file.  The client using this should care that it's an image.